R/tbl_baseline_chg.R

```diff
@@ -0,0 +1,257 @@
+#' Change from Baseline
+#'
+#' Typical use is tabulating changes from baseline
+#' measurement of an Analysis Variable.
+#' @inheritParams tbl_roche_summary
+#' @inheritParams gtsummary::add_overall
+#' @param analysis_variable (`string`)\cr
+#'  String identifying the analysis values. Default is `AVAL`.
+#' @param change_variable (`string`)\cr
+#'  String identifying the change from baseline values. Default is `CHG`.
+#' @param id (`string`)\cr
+#'  String identifying the unique subjects. Default is `USUBJID`.
+#' @param test_variable (`string`)\cr
+#'  String identifying the column containing lab test codes.
+#' @param test_cd (`string`)\cr
+#'  String identifying the lab test code to compute the change from baseline. This must be a value contained in `test_variable`.
+#' @param baseline_level (`string`)\cr
+#'  String identifying baseline level in the `visit` variable.
+#' @param header_label (`string`)\cr
+#'  String identifying the column header.
+#' @param denominator (`string`)\cr
+#'  Data set used to compute the header counts (typically `ADSL`).
+#' @param visit (`string`)\cr
+#'  String for the visit variable. Default is
+#'  `VISIT`.
+#' @param analysis_date (`string`)\cr
+#'  String identifying the visit or analysis sequence number. Default is
+#'  `VISITNUM`.
+#' @return a gtsummary table
+#' @name tbl_baseline_chg
+#'
+#' @examples
+#' library(dplyr, warn.conflicts = FALSE)
```

Comment:
Makes the example easier to read :)
```suggestion
#' theme_gtsummary_roche()
#' library(dplyr, warn.conflicts = FALSE)
```